EcpBtBot/MainClass.py

```diff
@@ -1,39 +1,42 @@
 
 import os
+import logging
 from .DialogFlow.API import *
 from .DialogFlow.DialogFlowResponse import *
 from .Conversational_Integration import *
```

Comment:
follow the alphabetical order in import (separating your code from external libraries though), it's cleaner

Reply:
ok